src/processTargets/marks/CursorStage.ts

```diff
@@ -0,0 +1,26 @@
+import { window } from "vscode";
+import { CursorMark, Target } from "../../typings/target.types";
+import { isReversed } from "../../util/selectionUtils";
+import { getTokenContext } from "../modifiers/scopeTypeStages/TokenStage";
+import { MarkStage } from "../PipelineStages.types";
+
+export default class implements MarkStage {
+  constructor(private modifier: CursorMark) {}
+
+  run(): Target[] {
+    if (window.activeTextEditor == null) {
+      return [];
+    }
+    return window.activeTextEditor.selections.map((selection) => {
```

Comment:
As above, why did you remove this info from the context object and get it here?  I'd prefer if the pipeline's only interactions with the environment are via context obj

Reply:
My opinion is that passing around objects for no good reason is something that should be avoided. Now it's only accessed when needed which I think is cleaner.

Reply:
You want me to add it back and I will?

Reply:
done